RELEASE_NOTES.md

```diff
@@ -2,7 +2,30 @@
 
 ## stable-1.15-1.9.0, stable-1.16-1.9.0
 
-* Traefik: fix metrics access and reporting ([#349](https://github.com/mesosphere/kubernetes-base-addons/pull/349), [@gracedo](https://github.com/gracedo))
+* Traefik 
+  * fix metrics access and reporting ([#349](https://github.com/mesosphere/kubernetes-base-addons/pull/349), [@gracedo](https://github.com/gracedo))
+* Elasticsearch
+  * default data nodes has been increased to 4 by default. ([#327](https://github.com/mesosphere/kubernetes-base-addons/pull/327), [@alejandroEsc](https://github.com/alejandroEsc))
+* Prometheus
+  * bump chart to v9.3.1 ([#444](https://github.com/mesosphere/kubernetes-base-addons/pull/444), [@samvantran](https://github.com/samvantran))
+    * refactor!: (breaking change) version 9 of the helm chart removes the existing `additionalScrapeConfigsExternal` in favor of `additionalScrapeConfigsSecret`. This change lets users specify the secret name and secret key to use for the additional scrape configuration of prometheus.
+    * feat: add ingress configuration for Thanos sidecar, enabling external access from a centralized thanos querier running in another cluster
+    * feat: add scrape timeout config to service monitor to avoid timeouts on slow kubelets
+    * feat: add docker checksum option to improve security for deployed containers
+    * feat: add option to disable availability rules
+    * feat: enable scraping /metrics/resource for kubelet service
+    * feat: enable namespace overrides
+    * feat: allow additional volumes and volumeMounts
+  * alertmanager:
+    * feat: add volume and volume mounts to spec
+    * feat: add support for serviceAccount.annotations
+  * grafana:
+    * feat: enable adding annotations to all default dashboard configmaps
+    * fix: add missing grafana annotations to k8s-coredns dashboard
+    * fix: reduced CPU utilization and time lag for code_verb:apiserver_request_total:increase30d scrape
+    * fix: invalid image pull policy for the admission webhook patch
+    * fix: alert "KubeNodeUnreachable" no longer fires on an autoscaling scale-down event
```

Comment:
this was something i added a few days ago and its a major version chart change. I dont think we should include it in the patch KBA version for 1.9 (that deploys on konvoy 1.4). Was the commit pulled into stable-1.x or is this just accidental release notes added?

Reply:
Yeah this the commit https://github.com/mesosphere/kubernetes-base-addons/pull/444.
@joejulian I can create a PR to `stable-1.x` with the serviceMonitor changes that is based on the previous revision.

Reply:
ok

Reply:
yeah let's put this only in KBA 2.x.

Reply:
https://github.com/mesosphere/kubernetes-base-addons/pull/476